index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav><img id="menupng" src="/assets/ic_menu.png"></nav>
```

Comment:
- You did a great job using a `nav` for your Hamburger button, but the image is not showing on your portfolio. Please consider fixing this and try: `./assests/ic_menu.png` I believe you are missing the `.` in your path 🩺.
![image](https://user-images.githubusercontent.com/43172164/198091927-2e2349dd-2af8-4817-b01c-e497b7447c56.png)

Reply:
Thanks for the suggestions, I have implemented these changes.